content/design-guidelines/backgrounds.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Backgrounds
+---
+
+## Backgrounds
```

Comment:
I think it's odd to jump from one heading to another, especially as they're the same.
```suggestion

```